src/live.asp.net/Services/FeedService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace live.asp.net.Services
+{
+    public class FeedService
+    {
+        private readonly IShowsService _showsService;
+        private readonly bool _disableCache;
+
+        public FeedService(IShowsService showsService, bool? disableCache)
+        {
+            _showsService = showsService;
+            _disableCache = disableCache ?? false;
+        }
+
+        public async Task<string> GetFeed()
+        {
+            var parent = new StringWriter();
+            var settings = new XmlWriterSettings { Indent = true };
+            using (var w = XmlWriter.Create(parent, settings))
+            {
+                w.WriteStartElement("feed", "http://www.w3.org/2005/Atom");
+
+                w.WriteElementString("id", "ASP.NET Community Standup");
+
+                w.WriteStartElement("link");
+                w.WriteAttributeString("type", "text/html");
+                w.WriteAttributeString("rel", "alternate");
+                w.WriteAttributeString("href", "https://live.asp.net");
+                w.WriteEndElement();
+
+                w.WriteElementString("title", "ASP.NET Community Standup Recorded Shows Feed");
+                w.WriteElementString("updated", DateTime.UtcNow.ToString());
```

Comment:
According to the [Atom spec](https://tools.ietf.org/html/rfc4287) dates should be formatted with [`yyyy-MM-ddTHH:mm:ssK`](https://tools.ietf.org/html/rfc3339)